lib/widgets/video_progress_widget.dart

```diff
@@ -35,88 +33,84 @@ class VideoProgressWidget extends StatelessWidget {
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
-                if (easelProvider.videoLoadingError.isNotEmpty)
-                  const SizedBox()
-                else
-                  easelProvider.isVideoLoading
-                      ? const SizedBox()
-                      : Row(
-                          children: [
-                            if (easelProvider.isVideoLoading)
-                              SizedBox(height: 22.h, width: 22.h, child: CircularProgressIndicator(strokeWidth: 2.w, color: EaselAppTheme.kBlack))
-                            else if (easelProvider.videoPlayerController.value.isPlaying)
-                              InkWell(
-                                onTap: easelProvider.pauseVideo,
-                                child: Icon(
-                                  Icons.pause,
-                                  color: darkMode ? EaselAppTheme.kWhite : EaselAppTheme.kDarkBlue,
-                                  size: 25.h,
-                                ),
-                              )
-                            else
-                              InkWell(
-                                onTap: easelProvider.playVideo,
-                                child: Icon(
-                                  Icons.play_arrow,
-                                  color: darkMode ? EaselAppTheme.kWhite : EaselAppTheme.kDarkBlue,
-                                  size: 25.h,
-                                ),
-                              ),
-                            SizedBox(width: 10.w),
-                            Expanded(
-                              child: VideoProgressIndicator(
-                                easelProvider.videoPlayerController,
-                                allowScrubbing: true,
-                                colors: VideoProgressColors(
-                                  backgroundColor: darkMode ? EaselAppTheme.kWhite : EaselAppTheme.kBlack,
-                                  playedColor: EaselAppTheme.kLightRed,
-                                  bufferedColor: EaselAppTheme.kWhite.withOpacity(0.7),
-                                ),
-                              ),
-                            ),
-                          ],
-                        ),
-                if (easelProvider.videoLoadingError.isNotEmpty || easelProvider.isVideoLoading)
-                  const SizedBox()
-                else
-                  SizedBox(
-                    height: 15.h,
-                    child: Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        padding: EdgeInsets.only(right: 30.w, bottom: 10.h, top: 10.h, left: 25.w),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            if (easelProvider.videoLoadingError.isNotEmpty)
+              const SizedBox()
```

Comment:
Why we are not showing the video loading error to use. 

there is no need for showing empty sized boxes.